code/datums/mutantraces.dm

```diff
@@ -1120,7 +1122,7 @@ TYPEINFO(/datum/mutantrace)
 		. = ..()
 
 	proc/set_head(var/head)
-		src.head = head
+		src.head_tracker = head
```

Comment:
Should add a line above 1125 that checks for the current `head_tracker` and, if it exists, sets its `linked_human` to `null`.